### ISSUE TITLE
All components have license = Proprietary

### ISSUE THREAD

**Issue by remi-filament** [acsone/weblate-odoo-component-generator#17]
You can see on any component that license = Proprietary, for instance here : https://translation.odoo-community.org/projects/sale-workflow-10-0/sale-workflow-10-0-portal_sale_personal_data_only/#information

I am not sure which license should be applied on translation but I guess it should be AGPL-3 as per the repo/modules, not Proprietary.

Will need to investigate how to update these and have new created components use correct license by default

**Comment by sbidoul** [comment]
The license is determined by the addon manifest. So AGPL would not be accurate. Don't we have a license comment or something that explains that. I'll need to look that up when at the PC.

**Comment by remi-filament** [comment]
Yes of course not all modules are AGPL-3, we probably would need to take the value from manifest when creating components to force it instead of default "Proprietary" value.
